src/Data/CAS/RocksDB.hs

```diff
@@ -314,15 +317,39 @@ newTable
     -> Codec v
     -> Codec k
     -> [B.ByteString]
+    -> IO (RocksDbTable k v)
+newTable db valCodec keyCodec namespace = do
+    table <- evaluate (unregisteredNewTable db valCodec keyCodec namespace)
+    tableInsert (tablesTable db) namespace ()
+    return table
+{-# INLINE newTable #-}
+
+-- | Create a new 'RocksDbTable' in the given 'RocksDb' without registering it with the "tables" table.
+--
+unregisteredNewTable 
+    :: HasCallStack 
+    => RocksDb
+    -> Codec v
+    -> Codec k
+    -> [B.ByteString]
     -> RocksDbTable k v
-newTable db valCodec keyCodec namespace
+unregisteredNewTable db valCodec keyCodec namespace
     | any (B8.any (\x -> x `elem` ['$', '%', '/'])) namespace
         = error $ "Data.CAS.RocksDb.newTable: invalid character in table namespace: " <> sshow namespace
     | otherwise
         = RocksDbTable valCodec keyCodec ns (_rocksDbHandle db)
   where
     ns = _rocksDbNamespace db <> "-" <> B.intercalate "/" namespace
-{-# INLINE newTable #-}
+{-# INLINE unregisteredNewTable #-}
+
+tablesTable
+    :: RocksDb
+    -> RocksDbTable [B.ByteString] ()
+tablesTable db = unregisteredNewTable 
+    db
+    (Codec (\() -> "") (\_ -> pure ())) 
+    (Codec (B.intercalate "/") (pure . B8.split '/'))
```

Comment:
This Codec doesn't roundtrip in general.

It is safe under the assumption that the keys are valid table names, but that isn't enforced. Maybe we would need a newtype wrapper that witnesses that fact?

Reply:
`tablesTable` only has a single user, so I think a newtype wrapper would not add any safety.